ovirt/provider.go

```diff
@@ -41,7 +41,6 @@ func Provider() terraform.ResourceProvider {
 			"headers": {
 				Type:        schema.TypeMap,
 				Optional:    true,
-				Default:     map[string]string{},
 				Description: "Additional headers to be added to each API call",
```

Comment:
For more precise, we may need to specify the type of `value` in map (it was my bad). So could you please help me add the definition of  `Elem` attribute here? Thank you.